README.md

```diff
@@ -59,66 +59,12 @@ If preCICE (the C++ library) was installed in a custom prefix, or only built but
 * `LIBRARY_PATH`, `LD_LIBRARY_PATH` to the library location, or `$prefix/lib`
 * `CPATH` either to the `src` directory or the `$prefix/include`
 
-The preCICE documentation provides more informaiton on [linking preCICE](https://precice.org/installation-linking.html).
+The preCICE documentation provides more information on [linking preCICE](https://precice.org/installation-linking.html).
 
 ### Using Spack
 
 You can also install the python language bindings for preCICE via Spack by installing the Spack package `py-pyprecice`. Refer to [our installation guide for preCICE via Spack](https://precice.org/installation-spack.html) for getting started with Spack.
 
-### Using setup.py
-
-#### preCICE system installs
-
-In this directory, execute:
-
-```bash
-$ python3 setup.py install --user
-```
-
-#### preCICE at custom location (setting PATHS)
-
-see above. Then run
-
-```bash
-$ python3 setup.py install --user
-```
-
-#### preCICE at custom location (explicit include path, library path)
-
-1. Install cython and other dependencies via pip3
-
-   ```bash
-   $ pip3 install --user setuptools wheel cython packaging numpy
-   ```
-
-2. Open terminal in this folder.
-3. Build the bindings
-
-   ```bash
-   $ python3 setup.py build_ext --include-dirs=$PRECICE_ROOT/src --library-dirs=$PRECICE_ROOT/build/last
-   ```
-
-   **Options:**
-   * `--include-dirs=`, default: `''`
-     Path to the headers of preCICE, point to the sources `$PRECICE_ROOT/src`, or the your custom install prefix `$prefix/include`.
-
-   **NOTES:**
-
-   * If you have built preCICE using CMake, you can pass the path to the CMake binary directory using `--library-dirs`.
-   * It is recommended to use preCICE as a shared library here.
-
-4. Install the bindings
-
-   ```bash
-   $ python3 setup.py install --user
-   ```
-
-5. Clean-up *optional*
-
-   ```bash
-   $ python3 setup.py clean --all
-   ```
-
```

Comment:
I think this does not belong into this PR.

Reply:
This is part of the setup.py deprecation. Should I split this?

Reply:
Yes. Let's do this in a different PR.